plugins/foreman_discovery/4.0/index.md

```diff
@@ -707,6 +695,59 @@ foreman-discovery-image git
 [repository](https://github.com/theforeman/foreman-discovery-image) and find
 the README for further instructions.
 
+## 5.3 PXE-less discovery
+
+It is possible to use Discovery image directly as a CDROM/DVDROM ISO that can
+be also transferred to hard drive or USB stick (dd command). In this case,
+automatic discovery process is not started and user select primary network
+interface interactively. On the couple of screens user configures network
+credentials (IPv4 supported at the moment), Foreman URL and type and fills user
+facts which are then uploaded to the server.
```

Comment:
Maybe also discuss the custom facts more.  "These custom facts can be used with pre-defined discovery rules to let users automatically build hosts."  And then include the authentication comment here